### DISCUSSION TITLE
Piler / Docker-Setup / import.sh

### DISCUSSION THREAD

**Discussion by andbez** [jsuto/piler#191] [Q&A] [answered]
Hi,

just figured out installing Piler on Docker. Everything works as expected (importing of 25000 E-Mails for example via IMAP), but after a few ours the Import does not work anymore. When I start den Script as piler user in the piler-container (yes, there are new mails (read and unread) in my IMAP-Mailbox), following output is written:
piler@624d63c668d1:/tmp$ /usr/libexec/piler/import.sh 
~/imap /tmp
~/imap ~/imap
Nothing to read from import table
piler@624d63c668d1:/tmp$ 

I had this issue several times in my multiple setup tries and thought it 

was working now. But unfortunately not :( Any hints, where to start analyzing this issue?

Thank you!
![Uploading Bildschirmfoto 2024-10-13 um 17.38.06.png…]()

**Comment by jsuto** [comment, marked as answer]
I assume you created an import job on the gui. Once the import job has run, imapfetch.py won't run it again, since it's completed.
If possible, then setup an smtp delivery to the archive, this is the preferred method to get new emails. If you can't or won't, then run imapfetch.py from a cron job directly. Once you have the initial import done, you may run something similar like below to get the emails since 2024-OCT-01. Just be sure to update this expression every day

```
imapfetch.py ... --date "SINCE \"01-Oct-2024\""
```
